httpd/handler.go

```diff
@@ -175,6 +179,127 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 	httpResults(w, results, pretty)
 }
 
+type Point struct {
```

Comment:
I don't see how BatchPoints would work here. All points being dumped don't share tags or timestamps. Also, I want to send small chunks off to write as they are read.